src/CompositeTilemap.ts

```diff
@@ -166,22 +166,22 @@ export class CompositeTilemap extends Container
     }
 
     /** Changes `animX`, `animCountX` of the last added tile. */
-    tileAnimX(offset: number, count: number): this
+    tileAnimX(offset: number, count: number, duration: number): this
```

Comment:
Duration should be an optional parameter.

Reply:
One think that's maybe better is to remove duration from here and give it its own setter method - tileAnimDurationFactor?